xapian-bindings/python3/Makefile.am

```diff
@@ -149,7 +212,23 @@ $(stamp): except.i doccomments.i fixup-swig-py3-wrapper
 
 -include xapian_wrap.d
 
-CLEANFILES += xapian_wrap.d $(stamp)
```

Comment:
It seems cleaner to leave this alone and just add a separate `CLEANFILES += ...` for the letor files below - that way all the core stuff is together followed by all the letor stuff.

Reply:
fixed in a fixup commit.